app/assets/javascripts/discourse/app/routes/app-route-map.js

```diff
@@ -16,11 +16,16 @@ export default function () {
     { path: "/t/:slug/:id", resetNamespace: true },
     function () {
       this.route("fromParams", { path: "/" });
+      // eslint-disable-next-line ember/routes-segments-snake-case
       this.route("fromParamsNear", { path: "/:nearPost" });
     }
   );
 
-  this.route("topicBySlugOrId", { path: "/t/:slugOrId", resetNamespace: true });
+  // eslint-disable-next-line ember/routes-segments-snake-case
+  this.route("topicBySlugOrId", {
+    path: "/t/:slugOrId",
+    resetNamespace: true,
+  });
```

Comment:
What would we need to do to resolve these? Update the route definitions somehow to avoid camelCase parameters?

Reply:
Turns out it's not used in many places so I can just fix it 👍